crates/cli/src/subcommands/publish.rs

```diff
@@ -189,6 +190,9 @@ pub async fn exec(config: Config, args: &ArgMatches) -> Result<(), anyhow::Error
             } else {
                 println!("{} database with identity: {}", op, database_identity);
             }
+            if let Some(update_summary) = update_summary {
+                println!("{}", update_summary);
```

Comment:
TODO: I guess I should check if the terminal supports ANSI color codes here, since the summary may include those.

Reply:
Is this TODO meant for this PR, or should it be a comment/ticket?

Reply:
I'm not sure that this is the best UX. It seems like these update summaries can be pretty large, and I don't think that most users are interested in reviewing them. Could we put this behind a CLI flag `--show-migration-summary` or something like that?

Reply:
Just a reminder to myself.

We could put these behind a flag, but then 99.9% of users will never see them. Our tools are already pretty chatty and I don't think this adds an overwhelming amount of information. It's good to tell the user when they're modifying their database.

It's particularly important to print an explanation when automatic migrations fail -- but, come to think of it, this PR hasn't implemented that, so I suppose I should do it as well, either here or in another PR.

Reply:
Is that reminder meant to be implemented in this PR? (i.e. should I hold off on reviewing?)

Reply:
For this PR yeah, I'll mark as draft until it's ready again.